data/types/device/status/status.go

```diff
@@ -26,9 +26,10 @@ func Names() []string {
 type Status struct {
 	device.Device `bson:",inline"`
 
-	Duration *int       `json:"duration,omitempty" bson:"duration,omitempty"`
-	Name     *string    `json:"status,omitempty" bson:"status,omitempty"`
-	Reason   *data.Blob `json:"reason,omitempty" bson:"reason,omitempty"`
+	Duration         *int       `json:"duration,omitempty" bson:"duration,omitempty"`
+	DurationExpected *int       `json:"expectedDuration,omitempty" bson:"expectedDuration,omitempty"`
```

Comment:
why not just `ExpectedDuration` as you keep on flipping the naming in other places

Reply:
See above.